src/smart-components/group/policy/add-policy/add-policy-wizard.js

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import { Wizard } from '@patternfly/react-core';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/';
+import { fetchGroupPolicies, fetchPolicy, createPolicy } from '../../../../redux/actions/policy-actions';
+import { fetchRoles } from '../../../../redux/actions/role-actions';
+import SummaryContent from './summary-content';
+import PolicyInformation from './policy-information';
+import PolicyContent from './policy-content';
+
+const AddPolicyWizard = ({
```

Comment:
Is there a way to get to this wizard in UI or is that for future PR?

Reply:
future PR